tree/ntuple/v7/src/RPageSinkBuf.cxx

```diff
@@ -22,6 +22,15 @@
 
 #include <algorithm>
 
+void ROOT::Experimental::Detail::RPageSinkBuf::RColumnBuf::DropBufferedPages()
+{
+   for (auto &bufPage : fBufferedPages) {
+      fCol.fColumn->GetPageSink()->ReleasePage(bufPage.fPage);
+   }
+   fBufferedPages.clear();
+   fSealedPages.clear();
```

Comment:
Could you add some text explaining (or pointing to the explanation) why the content of `fBufferedPages` needs to be released by not the content of `fSealedPages`?